packages/k2-alpine/src/index.ts

```diff
@@ -1,4 +1,8 @@
-export { useDripsyTheme as useTheme } from 'dripsy'
-export * from './components'
-export * from './theme/ThemeProvider'
-export * from './utils'
+import Constants from 'expo-constants'
+import { init as initStorybook } from './storybook'
+
+if (Constants.expoConfig?.extra?.storybook === true) {
+  initStorybook()
```

Comment:
`Constants.expoConfig?.extra?.storybook` is only true in expo go app, and I guess this is the easiest way to have separate entries for storybook app and package.